tutorials/using-xgboost/1.distributed-cpu.ipynb

```diff
@@ -0,0 +1,128 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Distributed XGBoost (CPU)\n",
+    "\n",
+    "Scaling out on AmlCompute is simple! The code from the previous notebook has been modified and adapted in [src/run.py](src/run.py). In particular, changes include:\n",
+    "\n",
+    "- import and initialize dask_mpi\n",
+    "- use argparse to allow for command line argument inputs\n",
+    "- mlflow logging \n",
+    "\n",
+    "The [environment.yml](environment.yml) contains the conda environment specification."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Get Workspace"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core import Workspace\n",
+    "\n",
+    "ws = Workspace.from_config()\n",
+    "ws"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Distributed Remotely\n",
+    "\n",
+    "Simply use ``MpiConfiguration`` with the desired node count."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core import ScriptRunConfig, Experiment, Environment\n",
+    "from azureml.core.runconfig import MpiConfiguration\n",
+    "\n",
+    "arguments = [\"--num_boost_round\", 100, \"--learning_rate\", 0.2, \"--gamma\", 0]\n",
```

Comment:
Split out things the user can set (like this but also especially the compute_target name) into a separate cell so user has clean vision of everything they can/need to set before running the training.